conans/cli/commands/search.py

```diff
@@ -5,38 +5,72 @@
 from conans.cli.command import conan_command, Extender
 
 
+def print_output(info, show_colors):
```

Comment:
If this is not used outside the ``search.py`` module, make it ``_print_output()``. We tend to do this by default, to identify at first glance what is the public interface of a module.

Reply:
Done